### PR TITLE
chore(deps): upgrade Faro to v1.4.2

### PR DESCRIPTION
**Pull request by codecapitano**
**What is this feature?**

Upgrade Faro to v 1.4.2

**Why do we need this feature?**

Newest features and fixes

**Who is this feature for?**

Devs and SREs

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -245,8 +245,8 @@
     "@grafana/data": "workspace:*",
     "@grafana/e2e-selectors": "workspace:*",
     "@grafana/experimental": "1.7.10",
-    "@grafana/faro-core": "^1.3.6",
-    "@grafana/faro-web-sdk": "^1.3.6",
+    "@grafana/faro-core": "^1.4.2",
+    "@grafana/faro-web-sdk": "^1.4.2",
```
Comment (tskarhed) [inline package.json:249]:
```suggestion
    "@grafana/faro-core": "1.4.2",
    "@grafana/faro-web-sdk": "1.4.2",
```
This should apply to the others too, I think

**Comment by tskarhed** [comment]
Please do a merge with main and update the dependencies again.

File: package.json (hunk 1 above)
Comment (tskarhed) [reply package.json:249]: Although this should already cover minor releases. Do we want `^` or `~` just because we develop this internally?

**Comment by ashharrison90** [comment]
we use renovate to manage our dependencies in core grafana. this has already been updated now.

for grafanistas: if you want to see what is due to be updated or manually trigger an update for a particular package, see the [dependency dashboard](https://github.com/grafana/grafana/issues/40378)

**Pull request closed without merging**